docs/source/conf.py

```diff
@@ -197,56 +197,64 @@
 # -- Options for LaTeX output ---------------------------------------------
 
 latex_elements = {
-# The paper size ('letterpaper' or 'a4paper').
-#'papersize': 'letterpaper',
-
-# The font size ('10pt', '11pt' or '12pt').
-#'pointsize': '10pt',
-
-# Additional stuff for the LaTeX preamble.
-#'preamble': '',
+    # The paper size ('letterpaper' or 'a4paper').
+    #'papersize': 'letterpaper',
+    # The font size ('10pt', '11pt' or '12pt').
+    #'pointsize': '10pt',
+    # Additional stuff for the LaTeX preamble.
+    #'preamble': '',
 }
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-  ('index', 'osBrain.tex', u'osBrain Documentation',
-   u'Open Sistemas de Información Internet S.L.', 'manual'),
+    (
+        'index',
+        'osBrain.tex',
+        u'osBrain Documentation',
```

Comment:
Now that we're at it, do we need the u-strings here? They look very... python2

Reply:
Probably not. I think this should be a separate issue anyways. We could update the whole `conf.py`, which is from 2014.